node_modules/oae-tests/lib/util.js

```diff
@@ -974,7 +974,7 @@ var createInitialTestConfig = module.exports.createInitialTestConfig = function(
 
     // The Cassandra connection config that should be used for unit tests, using
     // a custom keyspace for just the tests
-    config.cassandra.keyspace = 'oaeTest';
+    config.cassandra.keyspace = 'oaetest';
```

Comment:
I'd prefer changing it back if it's not important not to; it might be referenced from unexpected places in puppet-hilary etc.